ci/acceptance/run-dev.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
This file is unrelated - it is here to allow us to have a separate path in CI for testing changes in development